src/HHbbVV/postprocessing/PostProcessVBFZH_vars.txt

```diff
@@ -0,0 +1,418 @@
+('ak8FatJetHbb', 0)
```

Comment:
What is this file?

Reply:
It seems that I accidentally committed this file at some point. I should delete this.